src/html.ts

```diff
@@ -25,20 +26,36 @@ navigator.serviceWorker.register('${path}', { scope: '${options.scope}' })
 }`.replace(/\n/g, '')
 }
 
+export function checkForHtmlHead(html: string) {
+  if (!html.includes('</head>')) {
+    if (!html.includes('<body>')) {
+      console.warn([
+        '',
+        yellow('PWA WARNING:'),
+        '</head> and <body> not found in the html, the service worker and web manifest will not be injected.',
```

Comment:
replace this message with:
`</head> and <body> not found in the html, the service worker and the web manifest may not be injected.`